packages/request/tsup.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from "tsup";
+import { config } from "dotenv";
+
+const env = {};
+config({ processEnv: env });
+
+// TODO: Remove after confirmation and testing
+console.log("REQUEST PACKAGE ENV VARS", env);
```

Comment:
why add a whole config file for tsup here?

Reply:
I had to add it to be able to use environmental variables.